### DISCUSSION TITLE
404 page returns status 200

### DISCUSSION THREAD

**Discussion by dannysteenman** [vercel/vercel#5006] [Help] [answered]
If I try to request a non-existent page e.g.:
https://dannys-cloud.dsteenman.vercel.app/43

It returns my custom 404 page but the header shows https status 200 (checked with redirect path (chrome extension)).

<img width="479" alt="Screen Shot 2020-08-07 at 13 06 25" src="https://user-images.githubusercontent.com/15192660/89639995-e66c7c80-d8ae-11ea-96ad-7e07fb46766f.png">

This is my vercel.json
```
{
  "version": 2,
  "routes": [
    {
      "handle": "filesystem"
    },
    {
      "src": "/(.*)",
      "status": 404,
      "dest": "/404/index.html",
      "continue": true
    },
    {
      "src": "/assets/images/touch/(.*)",
      "status": 308,
      "headers": {
        "Location": "/assets/images/favicon/$1"
      },
      "continue": true
    },
    {
      "src": "/(.*)",
      "headers": {
        "X-Content-Type-Options": "nosniff",
        "X-Frame-Options": "DENY",
        "X-XSS-Protection": "1; mode=block",
        "Strict-Transport-Security": "max-age=63072000; includeSubDomains; preload",
        "Referrer-Policy": "no-referrer-when-downgrade",
        "Feature-Policy": "geolocation 'self'",
        "Content-Security-Policy": "default-src 'unsafe-inline' https: 'self' data:; upgrade-insecure-requests; frame-ancestors 'self'"
      }
    }
  ]
}
```

**Comment by styfle** [comment, marked as answer]
Looks like its returning 404 for me.

Also consider removing the route and instead emit a `404.html` file in your output directory.

https://vercel.com/docs/configuration#routes/advanced/custom-404

**Reply by dannysteenman** [reply]
Yes that's what I've done, so I guess you see the fixed version of my site now. But if you put the 404.html in another location and include your routing it won't work.

**Reply by dannysteenman** [reply]
Here is a build where it returns a 200: https://dannys-cloud-mmlil9j9x.vercel.app/pagedoesntexist

**Reply by styfle** [reply]
You need to remove `"continue": true`

```json
{
  "routes": [
    { "handle": "filesystem" },
    { "src": "/(.*)", "status": 404, "dest": "/404/index.html" }
  ]
}
```
